municipal_finance/static/javascript/table.js

```diff
@@ -425,6 +432,13 @@
           }
         }
       }
+
+      // highlighted rows
+      for (var h = 0; h < toHighlight.length; h++) {
```

Comment:
Agreed. In some places I've used underscore's ``_.each(array, function(a) { ... } );`` but it's not really any more readable than a for loop and is harder to debug because of the function.